apps/condo/pages/auth/change-password.tsx

```diff
@@ -1,152 +1,166 @@
-// @ts-nocheck
-/** @jsx jsx */
-import { css, jsx } from '@emotion/core'
-import { useState } from 'react'
-import { Button, Form, Input, Result, Typography, Row, Col } from 'antd'
-import Head from 'next/head'
+import { jsx } from '@emotion/core'
 import Router from 'next/router'
+
 import { useIntl } from '@core/next/intl'
-import { useMutation } from '@core/next/apollo'
-import { gql } from 'graphql-tag'
+import { Form, Input, Typography } from 'antd'
+import { Button } from '@condo/domains/common/components/Button'
+import AuthLayout from '@condo/domains/common/components/containers/BaseLayout/AuthLayout'
+import React, { useState } from 'react'
 
-import { TopMenuOnlyLayout } from '@condo/domains/common/components/containers/BaseLayout'
 import { getQueryParams } from '@condo/domains/common/utils/url.utils'
 import { runMutation } from '@condo/domains/common/utils/mutations.utils'
+import { useMutation } from '@core/next/apollo'
+import { gql } from 'graphql-tag'
+
+
+
+const INPUT_STYLE = { width: '273px' }
 
 const CHANGE_PASSWORD_WITH_TOKEN_MUTATION = gql`
     mutation changePasswordWithToken($token: String!, $password: String!) {
         status: changePasswordWithToken(token: $token, password: $password)
     }
 `
 
-const ChangePasswordForm = () => {
+const ChangePasswordPage = (): React.ReactElement => {
```

Comment:
React.FC

Reply:
```suggestion
const ChangePasswordPage:React.FC = () => {
```

Reply:
SKipped for now - add TODO - need to fix typescript validation